### ISSUE TITLE
Certificate invalid or not present, please choice

### ISSUE THREAD

**Issue by slowfruit** [deltaxflux/fluxion#417]
### Which version of fluxion are you using?

FLUX 0.24 (rev.  124) by  deltax

### What distribution of Linux (including the version) are you running it on?
kali linux 2016.2

### What wireless adapter are you using? Exact model and chipset, statements like "internal" and "was working before" are not helpful at all.
TP-LINK TL-WN821N (v4)

### Which is the driver for it?
no

### Does it support injection ( output of aireplay-ng -9)
???

### General description of your issue along with the steps to reproduce it
![u66](https://cloud.githubusercontent.com/assets/25937725/23183409/59215c94-f895-11e6-921e-5be16ac43cfb.jpg)

Certificate invalid or not present, please choice or create ssl certificate.

I expect that it is wrong written handshake tested on different networks and different methods of interception, all this error. Tell me what to do

Search for the string DIALOG_WEB_DIR_ENG="rtl" , and replace it with DIALOG_WEB_DIR_HE="rtl" ,
not fix my problem

**Comment by deltaxflux** [comment]
Use the create one option. This will happen every time because fluxion create every time an certificate

**Comment by slowfruit** [comment]
Good. But the problem is that when they come to the fake page and enter password, the password is not displayed in fluxion. Is not it because of the fact that shaking recorded incorrectly and the program can not be compared with the correct password handshake?

**Comment by deltaxflux** [comment]
Do you get attemps?

**Comment by slowfruit** [comment]
I entered the password 9 times in a fake page, but the program is not displayed attempts
![a3](https://cloud.githubusercontent.com/assets/25937725/23216934/b241fd5e-f928-11e6-803f-0d03983ef740.JPG)

**Comment by deltaxflux** [comment]
Q;
1. Which sites do you use ?
2. Do you have requirements installled ?
3. Which php version and lighttpd version do you use ?
4. Do you see something in the attempt log

**Comment by slowfruit** [comment]
1. I use russian fake ap
2. I did not understand the question
3. lighttpd/1.4.45    php 7.0.15-1

Good, working. Probably helped to update the program, even at the device under attack, I tried to go to yandex.ru (before there was another site) and it is not until the end of download fake up page. 

After the upgrade fluxion, every time you start offering upgrade to the latest version, how to fix it?
Thanks for the help!
